test/integration/targets/lookup_option_names/aliases

```diff
@@ -0,0 +1,2 @@
+shippable/posix/group4
```

Comment:
I recommend a slightly different test target name to avoid confusion, since `{plugin_type}_{plugin_name}` is the convention used to associate targets with specific plugins.

You could use `lookup-option-names` for example.